modules/fuse/fuse_impl.cc

```diff
@@ -261,37 +302,39 @@ int fs::fuse_readdir(const char* path, void* buf, fuse_fill_dir_t filler,
 }
 
 void* fs::fuse_init(struct fuse_conn_info* conn, struct fuse_config* cfg) {
-  VLOG(2) << "fuse: initfs with vineyard socket " << state.vineyard_socket;
+  LOG(INFO) << "fuse: initfs with vineyard socket " << state.vineyard_socket;
 
   state.client.reset(new vineyard::Client());
   state.client->Connect(state.vineyard_socket);
 
   fuse_apply_conn_info_opts(state.conn_opts, conn);
-  conn->max_read = conn->max_readahead;
+  // conn->max_read = conn->max_readahead;
 
   cfg->kernel_cache = 0;
   return NULL;
 }
 
 void fs::fuse_destroy(void* private_data) {
-  VLOG(2) << "fuse: destroy";
+  LOG(INFO) << "fuse: destroy";
 
   state.views.clear();
+  state.mutable_views.clear();
   state.client->Disconnect();
 }
 
 int fs::fuse_access(const char* path, int mode) {
-  VLOG(2) << "fuse: access " << path << " with mode " << mode;
+  LOG(INFO) << "fuse: access " << path << " with mode " << mode;
 
   return mode;
 }
 
 int fs::fuse_create(const char* path, mode_t mode, struct fuse_file_info*) {
-  VLOG(2) << "fuse: create " << path << " with mode " << mode;
+  LOG(INFO) << "fuse: create " << path << " with mode " << mode;
   if (state.mutable_views.find(path) != state.mutable_views.end()) {
     LOG(ERROR) << "fuse: create: file already exists" << path;
     return EEXIST;
   }
+  LOG(INFO) << "creating " << path;
   state.mutable_views.emplace(
       path, std::shared_ptr<arrow::BufferBuilder>(new arrow::BufferBuilder()));
   return 0;
```

Comment:
Revisit those `LOG(INFO)`s to `DLOG(INFO)` instead. Otherwise the logging would be a major overhead in a release build.

Reply:
updated

> Revisit those `LOG(INFO)`s to `DLOG(INFO)` instead. Otherwise the logging would be a major overhead in a release build.